level-ws.js

```diff
@@ -129,10 +125,14 @@ WriteStream.prototype.toString = function () {
   return 'LevelUP.WriteStream'
 }
 
-WriteStream.prototype.destroy = function () {
-  if (this._destroyed) return
+WriteStream.prototype._final = function (cb) {
+  this._buffer = null
+  this.writable = false
+  process.nextTick(cb)
```

Comment:
Tried calling `cb()` directly, which didn't work, so changed to `process.nextTick` but also doesn't help.

Reply:
The key here, I think, is to `db.batch` whatever is left in the buffer, then call the callback.

Reply:
So basically, call `_flush`.